### ISSUE TITLE
Use case for --disable-u2f / u2f_disabled

### ISSUE THREAD

**Issue by ruimarinho** [cevoaustralia/aws-google-auth#149]
Hi,

I have dug into the source code and I can't seem to find a reason for the `--disable-u2f` flag. Initially I thought it would simply not require the U2F libs to be installed, but then I noticed a dedicated variant which adds the extra deps.

Looking at the issues, I get a sense this was simply never removed when the package variant was introduced?

**Comment by stevemac007** [comment]
Yep - great pickup - I did believe this was still in use to stop the loading of the U2F libraries, but it seems that this is not the case.

Will flag this for removal in an upcoming release.
